pkl-core/pkl-core.gradle.kts

```diff
@@ -316,3 +255,24 @@ spotless {
     target(files("src/main/antlr/PklParser.g4", "src/main/antlr/PklLexer.g4"))
   }
 }
+
+private fun Test.configureTest() {
+  inputs.dir("src/test/files/LanguageSnippetTests/input")
+    .withPropertyName("languageSnippetTestsInput")
+    .withPathSensitivity(PathSensitivity.RELATIVE)
+  inputs.dir("src/test/files/LanguageSnippetTests/input-helper")
+    .withPropertyName("languageSnippetTestsInputHelper")
+    .withPathSensitivity(PathSensitivity.RELATIVE)
+  inputs.dir("src/test/files/LanguageSnippetTests/output")
+    .withPropertyName("languageSnippetTestsOutput")
+    .withPathSensitivity(PathSensitivity.RELATIVE)
+}
+
+private fun Test.configureExecutableTest(engineName: String) {
```

Comment:
Any reason why `testWindowsExecutableAmd64` doesn't call this function, just mac and linux?

Reply:
Good catch!